src/util/windowedstatistics.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+
+#include <QHash>
+#include <QList>
+#include <cmath>
+#include <memory>
+
+
+#include "util/assert.h"
+#include "util/circularbuffer.h"
+// to tell the msvs compiler about `isnan`
+#include "util/math.h"
+
+/// These classes are used to compute statistics descriptors
+/// of a series of tempo values and are called from beatutils
+
+class WindowedStatistics {
```

Comment:
```suggestion
/// of a series of tempo values and are called from BeatUtils
class WindowedStatistics {
```
No newline between documentation and the class declaration.